pyspeckit/spectrum/units.py

```diff
@@ -400,6 +400,12 @@ def refX_units(self):
     def refX_units(self, value):
         log.warn("'refX_units' is deprecated; please use 'refX_unit'", DeprecationWarning)
         self.refX_unit = value
+    
+    def __getattr__(self, name):
+        if name == 'refX_unit':
+            return self.refX.unit
+        else:
+            object.__getattribute__(self, name)
```

Comment:
Ahh, ok, I didn't appreciate that.  It would be good to include a comment on this, e.g.:
`# can't use getattr because it triggers infinite recursion`